include/ChessEngine.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+#include "ChessBoard.h"
+#include "Response.h"
+#include "Bishop.h"
+#include "King.h"
+#include "Knight.h"
+#include "Pawn.h"
+#include "Queen.h"
+#include "Rook.h"
+
+class ChessEngine
+{
+	ChessBoard* _chess_board;  ///< Pointer to the chess board object
```

Comment:
Because in the exercise instraction it is written that "nothing should be changed in the main, except for lines 28 to 31, where you must insert your code that returns the appropriate code" .
The only way to do this is by using singleton.